src/search.js

```diff
@@ -68,7 +68,14 @@ export function search(query, filters) {
                     attackObject.id = result.ref;
                     attackObject.score = result.score;
                     attackObject.matchData = result.matchData;
+                    if (attackObject.type == "subtechnique") {
+                        console.log(attackObject.name);
+                        let parentTechnique = result.ref.slice(0, result.ref.length - 4);
+                        parentTechnique = attackData[parentTechnique].name + ": ";
+                        if (!attackObject.name.includes(parentTechnique)) attackObject.name = parentTechnique + attackObject.name;
```

Comment:
I don't want to build our solution around updating the `attackData` object, because that's a global and making ad hoc changes is going get confusing somewhere down the road. (E.g. some code wants to iterate over subtechniques but it can't tell which subtechniques titles have been changed.)

A related problem is that the parent technique title won't show up as a search hit:

![Screenshot 2023-06-22 at 10 42 05 AM](https://github.com/center-for-threat-informed-defense/attack-powered-suit/assets/320904/55faec87-f63a-413f-b4ad-abfcc57240fe)

In this example I would expect the "interpreter" after "T1059.008" to be highlighted but it's not--that text wasn't there when we did the indexing. I'll post a comment on the main thread about an alternative approach.